From: Joel Selvaraj <foss@joelselvaraj.com>
Date: Sun, 16 Mar 2025 20:13:10 -0500
Subject: [PATCH] sdm845.config: enable ROHM BU64748 camera lens actuator

Used by Poco F1 main rear camera (Sony IMX363).

Signed-off-by: Joel Selvaraj <foss@joelselvaraj.com>
(cherry picked from commit abd0a0bd8881de361f53b44272952c7ec71478eb)
Link: https://gitlab.com/sdm845-mainline/linux
Signed-off-by: Richard Acayan <mailingradian@gmail.com>
---
 arch/arm64/configs/sdm845.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/sdm845.config b/arch/arm64/configs/sdm845.config
index ad7c71b..0a6e82b 100644
--- a/arch/arm64/configs/sdm845.config
+++ b/arch/arm64/configs/sdm845.config
@@ -28,6 +28,7 @@ CONFIG_TOUCHSCREEN_NOVATEK_NVT_TS=m
 CONFIG_QCOM_GPI_DMA=m
 CONFIG_SND_SOC_TAS2559=m
 CONFIG_VIDEO_IMX363=m
+CONFIG_VIDEO_BU64748=m
 
 # Samsung S9 SM-G9600(starqltechn)
 CONFIG_SND_SOC_MAX98512=y
